src/components/ToDoItem/ToDoItem.js

```diff
@@ -26,7 +36,6 @@ export const ToDoItem = (message) => {
   closeButton.classList.add('close-button');
 
   // METHODS
-
   const editButtonHoverOn = () => {
     editButton.style.opacity = 1;
```

Comment:
should be named handleEditButtonHoverOn, OR you do not actually wrap it inside another function - it is readable enough to have it just inside addEventListener

Reply:
renamed